promptsource/templates/sciq/templates.yaml

```diff
@@ -2,28 +2,30 @@ dataset: sciq
 templates:
   0af52ad2-2b12-4700-b664-cd26d2da6dc1: !Template
     answer_choices: null
-    answer_choices_key: distractor1 ||| distractor2 ||| distractor3 ||| correct_answer
+    answer_choices_key: '{{distractor1}} ||| {{distractor2}} ||| {{distractor3}} |||
+      {{correct_answer}}'
     id: 0af52ad2-2b12-4700-b664-cd26d2da6dc1
     jinja: 'Q: {{question}}
 
 
 
-      A:|||{{correct_answer}}
+      A:|||{{answer_choices[3]}}
 
       '
     metadata: !TemplateMetadata
       choices_in_prompt: false
       metrics:
       - Accuracy
-      original_task: false
+      original_task: true
     name: Direct Question (Closed Book)
     reference: ''
   15b0a989-84e4-4f1c-8ac1-12dbfa2ff42a: !Template
     answer_choices: null
-    answer_choices_key: distractor1 ||| distractor2 ||| distractor3 ||| correct_answer
+    answer_choices_key: '{{distractor1}} ||| {{distractor2}} ||| {{distractor3}} |||
+      {{correct_answer}}'
     id: 15b0a989-84e4-4f1c-8ac1-12dbfa2ff42a
-    jinja: "Q: {{question}}\n\n\n Choices:\n\n- {{ distractor1 }}\n\n- {{ distractor2\
-      \ }}\n\n- {{ distractor3 }}\n\n- {{ correct_answer }}\n\nA:|||{{correct_answer}}"
+    jinja: "Q: {{question}}\n\n\n Choices:\n\n- {{ answer_choices[0] }}\n\n- {{ answer_choices[1]\
+      \ }}\n\n- {{ answer_choices[2] }}\n\n- {{ answer_choices[3] }}\n\nA:|||{{answer_choices[3]}}"
```

Comment:
only thing i am worried about is that the order is deterministic in the sense that the last option is always the right one

Reply:
Yeah, it's annoying. I could add some code to randomize the order by calling choice on all permutations. Might not get to it until later this afternoon, if you want to unblock @craffel with the bug fixes. I could do a separate PR.

Reply:
ok let's do separate PR